src/styles/calendar.scss

```
@@ -0,0 +1,71 @@
+.calendar {
+  display: flex;
+  flex-direction: column;
+  gap: 20px;
+  width: $width-calendar;
+  font-family: Arial;
+
+  &__days {
+    display: flex;
+    justify-content: space-around;
+  }
+
+  &__day {
+    width: 100px;
+
+      @each $day, $number in $days {
+        &:nth-child(#{$number})::before {
+          content: "#{$day}";
+          text-transform: uppercase;
+          display: flex;
+          justify-content: center;
+          font-size: #{$font-title};
+        }
+      }
+  }
+
+  &__numbers {
+    display: flex;
+    flex-wrap: wrap;
+    gap: 1px;
+
+    @for $length from 28 through 31 {
+      &--month-length--#{$length} {
+        .calendar__number:nth-child(1n + #{$length + 1}) {
+          display: none;
+        }
+      }
+    }
+
+  @each $day, $margin in $daysOfBegining {
+    &--start-day--#{$day}
+      .calendar__number--1 {
+        margin-left: calc((#{$side-block} + 1px) * #{$margin});
+      }
+    }
+  }
+
+  &__number {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    align-content: center;
+    width: #{$side-block};
+    height: #{$side-block};
+    border: 1px solid black;
+    cursor: pointer;
+    transition: 0.5s;
```

```suggestion
    transition: transform 0.5s, background-color 0.5s;
```
And move 0.5s to the variable